server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -1725,7 +1726,8 @@ public IMNode getSeriesSchemasAndReadLockDevice(InsertPlan plan)
     String[] measurementList = plan.getMeasurements();
     IMeasurementMNode[] measurementMNodes = plan.getMeasurementMNodes();
 
-    // 1. get device node
+    // 1. get device node, set using template if accessed.
+    activateTemplateBeforeAccess(devicePath, measurementList);
     IMNode deviceMNode = getDeviceNodeWithAutoCreate(devicePath);
```

Comment:
If the device now is represented by template, this method maybe invalid.

Reply:
As fixed above, this methods no longer exists.